grid/src/List/ParentItem.js

```diff
@@ -15,16 +15,16 @@ const ParentItem = ({
     isParentRowOpen,

```

Comment:
I think we need to give types of props in this component.

Reply:
I have added props types